html/template/default/assets/scss/style.scss

```
@@ -40,6 +40,7 @@ p {
 @import "component/8.1.info";
 @import "component/8.2.banner";
 @import "component/9.1.mypage";
+@import "component/10.1.cookie_consent";
```

Stylelint (17.13.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)